File: fixtures/conformance/v1/python/service.py
from typing import overload

class Account:
    def __init__(self, account_id: str) -> None:
        self.id = account_id

@overload
def resolve(value: str) -> Account: ...

@overload
def resolve(value: int) -> Account: ...

def resolve(value: str | int) -> Account:
    return Account(str(value))

def exercise() -> Account:
    return resolve('acct-1')

File: fixtures/conformance/v1/typescript/service.ts
import { Account } from './models';

export namespace Billing {
  export function resolve(value: string): Account;
  export function resolve(value: number): Account;
  export function resolve(value: string | number): Account {
    return new Account(String(value));
  }
}

export function exercise(): Account {
  return Billing.resolve('acct-1');
}
